### PR TITLE
Replace `QtWebEngine` with `QtWebEngineQuick` plus some other misc fixes

### PR DESCRIPTION
**Pull request by Gela**
# Description

Replace `QtWebEngine` with `QtWebEngineQuick` and fix up a few other misc. issues.

## Type of change

- [x] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] Windows
- [ ] Android
- [ ] Linux
- [ ] macOS
- [ ] iOS

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/main.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include <QDir>
 #include <QQmlEngine>
 #ifdef QT_WEBVIEW_WEBENGINE_BACKEND
-#include <QtWebEngine>
+#include<QtWebEngineQuick>
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/AddItemsToPortal/main.cpp:20]:
```suggestion
#include <QtWebEngineQuick>
```

File: ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/main.cpp [hunk 2]
```diff
@@ -17,6 +17,10 @@
 #include <QDir>
 #include <QQmlEngine>
 
+#ifdef QT_WEBVIEW_WEBENGINE_BACKEND
+#include<QtWebEngineQuick>
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/main.cpp:21]:
```suggestion
#include <QtWebEngineQuick>
```

File: ArcGISRuntimeSDKQt_CppSamples/EditData/EditWithBranchVersioning/main.cpp [hunk 3]
```diff
@@ -22,6 +22,10 @@
 #include <QGuiApplication>
 #include <QQmlApplicationEngine>
 
+#ifdef QT_WEBVIEW_WEBENGINE_BACKEND
+#include<QtWebEngineQuick>
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/EditWithBranchVersioning/main.cpp:26]:
```suggestion
#include <QtWebEngineQuick>
```

File: ArcGISRuntimeSDKQt_CppSamples/Maps/CreateAndSaveMap/main.cpp [hunk 4]
```diff
@@ -17,6 +17,10 @@
 #include <QDir>
 #include <QQmlEngine>
 
+#ifdef QT_WEBVIEW_WEBENGINE_BACKEND
+#include<QtWebEngineQuick>
```
Comment (JamesMBallard) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/CreateAndSaveMap/main.cpp:21]:
```suggestion
#include <QtWebEngineQuick>
```

**Review by JamesMBallard — COMMENTED** [review]
A few minor formatting things to review. Also, there are lots of unrelated changes.

**Comment by Gela** [comment]
@JamesMBallard Thanks for reviewing. All those other changes are intentional, body of the PR mentions "fix up a few other misc. issues." but I noticed the PR title might be misleading. I will update that as well. Please review/approve if everything looks good

**Review by JamesMBallard — APPROVED** [review]

**Pull request merged**